deploy/dev/reload.sh

```diff
@@ -3,32 +3,49 @@ set -xeuo pipefail
 
 . utils.sh
 
-# Script for making it easy to make a change locally and redeploy
-pushd ..
-  ./bin/build
-popd
+main() {
+  # Clean-up previous run
+  helm delete $(helm ls -aq)
+  $cli_with_timeout "delete deployment test-env --ignore-not-found=true"
 
-set_namespace $APP_NAMESPACE_NAME
+  pushd ..
+    ./bin/build
+  popd
 
-docker tag "secrets-provider-for-k8s:dev" "${APP_NAMESPACE_NAME}/secrets-provider"
+  set_namespace $APP_NAMESPACE_NAME
 
-selector="role=follower"
-cert_location="/opt/conjur/etc/ssl/conjur.pem"
-if [ "$CONJUR_DEPLOYMENT" = "oss" ]; then
-  selector="app=conjur-cli"
-  cert_location="/root/conjur-${CONJUR_ACCOUNT}.pem"
-fi
+  docker tag "secrets-provider-for-k8s:dev" "${APP_NAMESPACE_NAME}/secrets-provider"
 
-conjur_pod_name=$($cli_with_timeout get pods --selector=$selector --namespace $CONJUR_NAMESPACE_NAME --no-headers | awk '{ print $1 }' | head -1)
-ssl_cert=$($cli_with_timeout "exec ${conjur_pod_name} --namespace $CONJUR_NAMESPACE_NAME cat $cert_location")
+  if [ "${DEV_HELM}" = "true" ]; then
+    setup_helm_environment
 
-export CONJUR_SSL_CERTIFICATE=$ssl_cert
+    export IMAGE_PULL_POLICY=Never
 
-export CONFIG_DIR="$PWD/config/k8s"
-if [[ "$PLATFORM" = "openshift" ]]; then
-  export CONFIG_DIR="$PWD/config/openshift"
-fi
+    deploy_chart
 
-$cli_with_timeout "delete deployment init-env --ignore-not-found=true"
+    deploy_helm_app
+  else
+    selector="role=follower"
+    cert_location="/opt/conjur/etc/ssl/conjur.pem"
+    if [ "$CONJUR_DEPLOYMENT" = "oss" ]; then
+      selector="app=conjur-cli"
+      cert_location="/root/conjur-${CONJUR_ACCOUNT}.pem"
+    fi
 
-deploy_env
+    conjur_pod_name=$($cli_with_timeout get pods --selector=$selector --namespace $CONJUR_NAMESPACE_NAME --no-headers | awk '{ print $1 }' | head -1)
+    ssl_cert=$($cli_with_timeout "exec ${conjur_pod_name} --namespace $CONJUR_NAMESPACE_NAME cat $cert_location")
+
+    export CONJUR_SSL_CERTIFICATE=$ssl_cert
+
+    export CONFIG_DIR="$PWD/config/k8s"
+    if [[ "$PLATFORM" = "openshift" ]]; then
+      export CONFIG_DIR="$PWD/config/openshift"
+    fi
```

Comment:
this seems to be done a couple of times throughout the code. how about moving it to one of the top scripts and doing it once?

Reply:
There are some that have different paths so those that have the same I will put in utils

Reply:
it's not really a util but more of a general thing you do.

Reply:
a lot of our scripts have `set -euxo pipefail` so we are unsetting the env variables anyway.